content/4_Gold/Combinatorics.problems.json

```diff
@@ -78,16 +78,27 @@
       }
     },
     {
-      "uniqueId": "cf-888D",
+      "uniqueId": "cf-732D",
       "name": "AquaMoon and Chess",
       "url": "https://codeforces.com/contest/1546/problem/D",
       "source": "CF",
       "difficulty": "Normal",
       "isStarred": false,
       "tags": ["Combinatorics"],
       "solutionMetadata": {
-        "kind": "autogen-label-from-site",
-        "site": "CF"
+        "kind": "internal"
+      }
+    },
+    {
+      "uniqueID:" "cf-888D",
+      "name": "Almost Identity Permutations",
+      "url": "https://codeforces.com/problemset/problem/888/D",
+      "source": "CF",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Combinatorics"],
+      "solutionMetadata": {
+        "kind": "internal",
       }
```

Comment:
isn't this problem a bit too easy to be in gold?
it was initially in there but we removed it i think

Reply:
I had seen the problem in the guide before so I did it. I didn't realized it was removed because it was too easy.
Should I do a different problem or switch the difficulty tag to 'Easy'?

Reply:
@ryanchou-dev @envyaims what yall think

Reply:
@SansPapyrus683 its a decent example of derangement. probably deserves easy since its kinda standard

Reply:
wb "very easy"

Reply:
nah

Reply:
why not

Reply:
no way its easier than those cses problems like come on

Reply:
https://codeforces.com/contest/888/submission/158112445
this is a python solution. it's like 10 lines of code.

Reply:
python code length isnt a good indicator of a difficulty of a problem lol...

Reply:
the point is, the solution is extremely simple

Reply:
it's ur choice then  🤡

Reply:
no clowns

Reply:
i think it's tolerable as like a very easy problem. iy don't know anything abt it then it takes a bit of thinking

Reply:
I think if we put this problem as 'very easy', we should also put the first two CSES problems as very as well because they are easier.